apps/web/pages/api/book/event.ts

```diff
@@ -375,7 +377,7 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
         title: evt.title,
         startTime: dayjs(evt.startTime).toDate(),
         endTime: dayjs(evt.endTime).toDate(),
-        description: evt.description,
+        description: evt.additionalNotes,
```

Comment:
Shouldn't this be: ?
```suggestion
        additionalNotes: evt.additionalNotes,
```

Reply:
No because the booking description is the addiitonal notes. I.E From a booking model perspective we don't care about the description from the eventType